src/database/schema/UserSchema.ts

```diff
@@ -182,14 +182,14 @@ class UserSchema implements BaseDbSchema {
     }
   };
 
-  isUserExists = async (
+  static isUserExists = async (
     db: SQLiteDatabase,
     userId: string,
     channelId: string,
     transaction?: Transaction
   ): Promise<boolean> => {
     try {
-      const query = `SELECT * FROM ${UserSchema.getTableName()} WHERE user_id = ? AND channel_id = ? LIMIT 1`;
+      const query = `SELECT user_id FROM ${UserSchema.getTableName()} WHERE user_id = ? AND channel_id = ? LIMIT 1`;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on optimizing the SQL query by selecting only the `user_id` instead of all columns. However, you've changed the method from an instance method to a static method. If this is intentional and you've updated all usages of this method accordingly, then it's fine. Otherwise, it might cause issues where this method is called on an instance of `UserSchema`.
```diff
-  isUserExists = async (
+  static isUserExists = async (
```
Also, remember to update the function documentation (if any) to reflect this change.


<!-- This is an auto-generated comment by OSS CodeRabbit -->